### PR TITLE
Popover menu

### PR DESCRIPTION
**Pull request by affordances**
Screen Shot 2022-08-26 at 3 44 35 PM

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx [hunk 1]
```diff
@@ -78,6 +78,129 @@ export class ComponentShowcase implements m.ClassComponent {
             })
           }
         /> */}
+        <h1>Popover Menu</h1>
+        <div class="basic-gallery">
+          <CWPopoverMenu
```
Comment (zakhap) [inline packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx:83]: @jnaviask @gdjohnson @affordances 
do we like this syntax for filling out the menu, an array of objects? 
or should it be formatted more like: 
```
<CWPopoverMenu 
  trigger={}
>
   <MenuItem label='' icon='' />
   <MenuItem type='divider' />
<CWPopoverMenu />

```

Comment (jnaviask) [reply packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx:83]: No strong preference for me. I'm not sure of best practices in tsx.

**Comment by zakhap** [comment]
Screen Shot 2022-08-26 at 4 27 05 PM

I'm getting the error on repeated clicking of the triple dot button on the component showcase page.

**Review by zakhap — APPROVED** [review]
Ok. Before this component gets used anywhere in production, I would like this null error message in the console to be fixed.

**Pull request merged**